.idea/misc.xml

```diff
@@ -22,9 +22,8 @@
       </list>
     </option>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_23" default="true" project-jdk-name="temurin-23" project-jdk-type="JavaSDK" />
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_22" project-jdk-name="temurin-23" project-jdk-type="JavaSDK" />
```

Comment:
Why not set it to `JDK_X` which afaik means "SDK default"?

Reply:
I am also ok with `JDK_X` i just need something my ide accepts.

Reply:
JDK_X means experimental features, and when a new JDK is used, which IntelliJ is unaware of, it switches automatically to `JDK_X`, see https://github.com/crate/crate/pull/16759